frontend/src/metabase/dashboard/containers/CopyDashboardForm.tsx

```diff
@@ -155,5 +154,4 @@ function CopyDashboardForm({
   );
 }
 
-export const CopyDashboardFormConnected =
-  _.compose(withRouter)(CopyDashboardForm);
+export const CopyDashboardFormConnected = withRouter(CopyDashboardForm);
```

Comment:
same fix as in `enterprise/frontend/src/embedding-sdk/components/public/CreateDashboardModal/CreateDashboardModal.tsx`, compose doesn't work well with generics, or maybe just `withRouter` specifically.